### ISSUE TITLE
Linux Install Fails

### ISSUE THREAD

**Issue by glrramos** [surrealdb/install.surrealdb.com#7]
Linux Installation with curl -k --proto '=https' --tlsv1.2 -sSf https://install.surrealdb.com | sh fails with error:

"Fetching the latest database version..."
"Error: could not fetch the latest SurrealDB version number"

![image](https://github.com/surrealdb/install.surrealdb.com/assets/14102079/8203a403-177f-46da-b866-cd8654f69df0)

**Comment by naisofly** [comment]
@glrramos thanks for bringing this up!

Have you tried running the command `curl -sSf https://install.surrealdb.com | sh` from [here](https://surrealdb.com/install)?

I'm looking into the inconsistencies with installation commands on our site :eye:

**Comment by glrramos** [comment]
I am using the command curl -sSfk https://install.surrealdb.com | sh to disable certificate verification.

![image](https://github.com/surrealdb/install.surrealdb.com/assets/14102079/d248ce0f-5c0a-4936-926c-412bba96b2cb)

**Comment by glrramos** [comment]
Yes, the only change I did is to use the -k option in the curl to disable
certificate verification.

[image: image.png]

Em ter., 1 de ago. de 2023 às 08:47, Naiyarah Hussain <
***@***.***> escreveu:
